SU-SWS/stanford_capx
# Hsdo 325

# Ready For Review - Do not merge

Multiple entity importing with entity references functionality.

Review:
1. Read over code changes for any glaring issues or poor choices

To Test: (Using JSA)
1. Install and connect to API
2. Create an importer and mapper for stanford_person (Use BUWS org and susskind sunet)
3. Create an importer and mapper for stanford_publication (same options as above)
4. On the publication mapper page for the entity reference field select the person importer
5. Ensure that both importers import content and that the entity reference is set
6. Delete and uninstall everything.

For the publications mapper you can use the following configuration:
![screen shot 2016-05-24 at 5 21 50 pm](https://cloud.githubusercontent.com/assets/550602/15554041/faa6e20a-2276-11e6-8f09-f0a95a8b7597.png)
